packages/synapse-react-client/src/components/EntityTreeTable/components/EntityNameCell.tsx

```diff
@@ -0,0 +1,57 @@
+import { CellContext } from '@tanstack/react-table'
+import { EntityBundleRow } from '../EntityTreeTable'
+import { Box } from '@mui/system'
+import { IconButton } from '@mui/material'
+import { ChevronRight, ExpandMore } from '@mui/icons-material'
+import { useEntityTreeTableContext } from './EntityTreeTableContext'
+import { EntityLink } from '@/components/EntityLink'
+import { SynapseSpinner } from '@/components/LoadingScreen/LoadingScreen'
+
+export const NameCell: React.FC<CellContext<EntityBundleRow, unknown>> = ({
+  row,
+}) => {
+  const { expanded, loadingIds, handleToggleExpanded, entityIdClicked } =
+    useEntityTreeTableContext()
+  const { entityHeader, depth, isLeaf } = row.original
+  const isExpanded = !!expanded[entityHeader.id]
+  const hasChildren = !isLeaf
+  return (
+    <Box sx={{ display: 'flex', alignItems: 'center', pl: depth * 2 }}>
```

Comment:
This `depth` factor is used in both `AutoLoadMore` and here, except in `AutoLoadMore` you are using pixels (16px), and here you are using MUI's spacing units (2 * 8px per unit).

I suggest refactoring the depth factor to be a shared constant and be consistent with using MUI spacing or px.

Reply:
reusing shared constant (and px only)